Cli documentation (#509)
opensearch-project/opensearch-py-ml: docs/source/cli/predict_model.rst

        Enter the path to save the output information, or press Enter to save it in the current directory [/Documents/cli/output.yml]: 

        Output information saved successfully to /Documents/cli/output.yml

* Direct model prediction with all parameters:

    Command:

    ``opensearch-ml model predict --modelId 'model123' --body '{"parameters": {"input": ["hello world", "how are you"]}}'``

    Sample response:
    
    .. code-block:: plaintext

        Starting model prediction...

        Setup configuration loaded successfully from /Documents/cli/setup_config.yml
        Predict Response: {"inference_results": [{"output": [{"name": "sentence_embedding", "data_type": "FLOAT32", "shape": [1536], "data": [-0.016099498, 0.001368687, -0.019484723, -0.033694793, -0.026005873, 0.0076758, -0.0..."status_code": 200}]}

        Successfully predict the model.
        Do you want to save the full prediction output? (yes/no): yes

        Enter the path to save the output information, or press Enter to save it in the current directory [/Documents/cli/output.yml]: 

        Output information saved successfully to /Documents/cli/output.yml

Output YAML file
~~~~~~~~~~~~~~~~

> After successfully predict a model, the CLI saves important information about the model prediction in an output YAML file. This file contains details that may be needed for future operations or reference. Here's an example of what the output YAML file might look like:

Reviewer: ```suggestion
After successfully running a model prediction, the CLI saves important information about the prediction in an output YAML file. This file contains details that may be needed for future operations or reference. The output YAML file appears as follows:
```
